lib/routers/profile/person.js

```diff
@@ -184,7 +191,9 @@ function getStatus(pils) {
 const getPil = (req, res, next) => {
   const { pil, trainingPils, pilLicenceNumber } = req.profile;
 
-  if (!pilLicenceNumber) {
+  const activeTrainingPils = trainingPils.filter(p => p.status === 'active');
```

Comment:
Is `trainingPils` guaranteed to be an array? (i.e. not `null` or `undefined`)

Reply:
I see this has moved up in the diff rather being actually added, so I'll assume yes.